CONTRIBUTING.md

```diff
@@ -0,0 +1,84 @@
+Guidelines for contributing to Hibernate Search
+====
+Contributions from the community are essential in keeping Hibernate Search strong and successful.
+This guide focuses on how to contribute back to Hibernate Search using GitHub pull requests.
+If you need help with cloning, compiling or setting the project up in an IDE please refer to
+[this wiki document](https://developer.jboss.org/wiki/ContributingtoHibernateSearch).
```

Comment:
Looks like the wiki is severely outdated (JDK7 and suggested Maven versions won't work)

The new place is:
 - http://hibernate.org/search/contribute/

although, it's missing some details like the JDK version.

We should update the website, and but a clear banner on the outdate wiki to point to the website?